src/main/main.ts

```diff
@@ -75,9 +75,10 @@ const createWindow = async () => {
     height: 728,
     icon: getAssetPath('icon.png'),
     webPreferences: {
-      preload: app.isPackaged
```

Comment:
this will break in cases where the app is running with a prod build but hasn't been packaged

Reply:
we could try testing against `process.env.NODE_ENV === 'test'`